src/inference/src/cpp_interfaces/interface/ie_iinfer_request_internal.cpp

```diff
@@ -85,6 +83,22 @@ std::map<std::string, InferenceEngineProfileInfo> IInferRequestInternal::GetPerf
     IE_THROW(NotImplemented);
 }
 
+std::shared_ptr<const ov::Node> IInferRequestInternal::findInputByNodeName(const std::string& name) const {
+    for (const auto& input : GetInputs()) {
+        if (input->get_friendly_name() == name)
+            return input;
+    }
+    return nullptr;
```

Comment:
Shouldn't we throw an error if no input found with such a name?

Reply:
No we shouldn't old API can work without inputs.